### ISSUE TITLE
[Core] Miranda NG Options menu sometimes becomes unusable and cannot be closed

### ISSUE THREAD

**Issue by LittleVulpix** [miranda-ng/miranda-ng#710]
In currently unexplored/undocumented scenarios (but not completely uncommon!), the Miranda NG options menu will become stuck. It will not allow you to select any of the options from the left column, nor will it allow to be closed.

Upon restart, miranda will sometimes produce 1-4 crashdumps which all list crashdumper as the cuplrit (but it's more likely it crashed trying to report the actual miranda issue).

![issue](https://i.imgur.com/FRz49BY.gif)

Example of a dump below:

```
Miranda Crash Report from 12 Feb 2017 2:51:52. Crash Dumper v.0.0.5.2
Exception: Access Violation at address 156EB97D. DEP at address 156EB97D.

Stack Trace:
---------------------------------------------------------------
156EB97D ((module-name not available) 00000000): (filename not available) (0): (function-name not available)
7617D2B3 (USER32 76150000): (filename not available) (0): SetManipulationInputTarget
7615E88A (USER32 76150000): (filename not available) (0): DispatchMessageW
7615E1E4 (USER32 76150000): (filename not available) (0): DispatchMessageW
7615DFA0 (USER32 76150000): (filename not available) (0): DispatchMessageW
121A543E (mir_app 12180000): c:\sources\miranda_ng_compilation\src\mir_app\src\miranda.cpp (331): mir_main
00401999 (Miranda32 00400000): c:\sources\miranda_ng_compilation\src\miranda32\src\miranda.cpp (65): wWinMain
004011B2 (Miranda32 00400000): f:\dd\vctools\crt_bld\self_x86\crt\src\crtexe.c (547): __tmainCRTStartup
770762C4 (KERNEL32 77060000): (filename not available) (0): BaseThreadInitThunk
77C20FD9 (ntdll 77BC0000): (filename not available) (0): RtlSubscribeWnfStateChangeNotification
77C20FA4 (ntdll 77BC0000): (filename not available) (0): RtlSubscribeWnfStateChangeNotification

CPU: Intel(R) Pentium(R) CPU J4205 @ 1.50GHz [Intel64 Family 6 Model 92 Stepping 9] [DEP Enabled] [4 CPUs]
Installed RAM: 15781 MBytes
Microsoft Windows 10 Professional Edition, 64-bit (build 14393)
Internet Explorer: 9.11.14393.0 (build 914393)
Administrator privileges: Yes
OS Languages: (UI | Locale (User/System)) : English/English | English/English

Miranda NG Version: 0.95.6 alpha build #17672 [running inside WOW64]
Build time: 11 Feb 2017 4:04:32
Language pack: No language pack installed
Service Mode: No

Active Plugins (4):
¤ AdvaImg.dll v.0.11.0.2 [2 Jan 2017 4:01:28] - Miranda image services 
¤ Clist_modern.dll v.0.9.1.5 [18 Jan 2017 4:01:52] - Modern contact list 
¤ CrashDumper.dll v.0.0.5.2 [13 Jan 2017 4:01:54] - Crash dumper 
¤ Dbx_mmap.dll v.0.95.3.1 [11 Feb 2017 4:00:14] - Miranda NG mmap database driver 
```

**Comment by georgehazan** [comment]
well, now I understand what concretely happens, but I dunno why :-)

**Comment by LittleVulpix** [comment]
:D so share with us, what is happening? It's so random too.

**Comment by dartraiden** [comment]
Seems like this fixed in 960a20dcd916e05fde9fb67cd42ad7f75f85a1f9
